USER-INTERFACE-INTERFACE.md

```diff
@@ -560,18 +561,31 @@ of payments that have not yet been confirmed. Only cumulative account balances a
 ```
 "payload": {
     "dbPassword": <string>,
-    "mnemonicPhraseSize": <number>,
-    "mnemonicPhraseLanguage": <string>,
-    "mnemonicPassphraseOpt": <optional string>,
-    "consumingDerivationPath": <string>,
-    "earningDerivationPath": <string>
+    "seedSpecOpt": {
+        "mnemonicPhraseSize": <number>,
+        "mnemonicPhraseLanguage": <string>,
+        "mnemonicPassphraseOpt": <optional string>
+    },
+    "consumingDerivationPathOpt": <optional string>,
+    "earningDerivationPathOpt": <optional string>
 }
 ```
 ##### Description:
-This message directs the Node to generate a pair of wallets and report their mnemonic phrase and their addresses
-back to the UI. If the database already contains a wallet pair, the wallet generation will fail.
+This message directs the Node to generate a pair of wallets and report their vital statistic back to the UI. 
+If the database already contains a wallet pair, the wallet generation will fail.
+
+Wallets can be generated in two ways:
```

Comment:
@dnwiebe reading through this a few times, it's unclear to me if the user will see the mnemonic seed words output if they run the generateWallets request.

In either of the two generation ways (using derivation paths, or at random), its clear that the consuming and earning wallets and private keys are output to the user.
But does the Node retain the original functionality that it will display (one time only) the seed words generated by the BIP39 algorithm?
Because even if the user writes down the private keys, they should be retaining the seed words as well for security overall, since they were used to derive one/both wallets, depending on the user choices (many users will opt for our default user experience which is simply one wallet for earning and consuming)